test-result-summary-client/src/Build/TestTable.jsx

```diff
@@ -71,13 +70,28 @@ export default class TestTable extends Component {
                 <Link to={{ pathname: '/deepHistory', search: params({ testId }) }}>
                     Deep History
                 </Link>
-                <span className="ant-divider" />
-                <a href={issueUrl} target="_blank" rel="noopener noreferrer">Possible Issues</a>
-
+                {possibleIssues(value)}
                 {gitIssue(row)}
             </span>
         }
 
+        const possibleIssues = (value) => {
+            const { testName } = value;
+            
+            let issueUrl = `https://github.com/AdoptOpenJDK/openjdk-tests/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+${testName}`;
+            if (buildData) {

```

Comment:
If there is no buildData, default `issueUrl ` to AdoptOpenJDK will cause inconsistent behavior. For example, we do not have buildData during search, but it does not mean the `buildName ` is not j9 or ibm. 

Please move line 81 and return inside of ` if (buildData)`.

Reply:
issueUrl and return have been moved into ```if (buildData)``` as suggested, thanks.